rest-api/src/main/resources/definitions/weekly_adherence_report.yml

```diff
@@ -28,11 +29,11 @@ properties:
         x-nullable: false
     byDayEntries:
         type: object
-        description: A map of activities to be performed at specific days since the event occurred. The map is sparse (days without activities do not have entries). The keys are string integers that correspond to the day on which activities occur, and the value is a detailed description of the activities to be performed on that day.
+        description: A map of activities to be performed at specific days since the event occurred. The keys are string integers that correspond to the day on which activities occur, and the value is list of descriptions of the activities to be performed on that day. The records in the list are in the same position as their row descriptors in the `rows` array of this report (that his, first row record describes the first entry in the list that is the value of this map). 
```

Comment:
Minor edits:
and the value is list of descriptions -> and the value is a list of descriptions
this report (that his, -> this report (that is,

Reply:
Updated